src/App.jsx

```diff
@@ -5,14 +5,13 @@ import { PersistGate } from "redux-persist/integration/react";
 import { BrowserRouter as Router, Routes } from "react-router-dom";
 import generateRoutes from "./utils/generateRoutes";
 import routes from "./routes/index";
-import PageOne from "./component/PageOne";
 
 function App() {
   return (
     <>
       <Provider store={store}>
         <PersistGate loading={null} persistor={persistor}>
-          <PageOne />
+          {/* <PageOne /> */}
```

Comment:
remove this instead of commenting . Don't have commented code inside the file unless super necessary

Reply:
Okay, I have removed it.